src/issuance.rs

```diff
@@ -652,6 +632,27 @@ mod tests {
         );
     }
 
+    fn setup_params() -> (
+        OsRng,
+        IssuanceAuthorizingKey,
+        IssuanceValidatingKey,
+        Address,
+        [u8; 32],
+    ) {
+        let mut rng = OsRng;
+        let sk = SpendingKey::random(&mut rng);
+        let isk: IssuanceAuthorizingKey = (&sk).into();
+        let ik: IssuanceValidatingKey = (&isk).into();
+
+        let fvk = FullViewingKey::from(&sk);
+        let recipient = fvk.address_at(0u32, Scope::External);
```

Comment:
Generally, the recipient is different from the sender. Here it has no effect but this function might be used elsewhere later so let's fix it.

Reply:
Actually, I didn't change `setup_params` function, I moved it down when I re-ordered functions in `tests` so it git shows it in red, which is confusing. I will return it to the top (which is more proper place for it as it is called from `setup_verify_supply_test_params`).

Anyway, I can fix it too - do you mean we need to return sender from `setup_params`?

Reply:
Yes, please generate the `recipient` from a fresh `sk`. If other tests break then abort.

Reply:
Fixed it this way:

```rust
        let sk = SpendingKey::random(&mut rng);
        let isk: IssuanceAuthorizingKey = (&sk).into();
        let ik: IssuanceValidatingKey = (&isk).into();

        let fvk = FullViewingKey::from(&SpendingKey::random(&mut rng));
        let recipient = fvk.address_at(0u32, Scope::External);
```

Local tests passed, CI tests are still running now.